@uportal/api-template-vue/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
Gitignore isn't needed, top level ignore will take care of this.

Reply:
The Vue CLI generates it -- that's why you're seeing them in PRs.